Starting the blog post introducing Marcos and Lukas (#539)
ropensci/roweb3: content/blog/2023-04-18-champions-program-teams/index.md

---
title: "rOpenSci Champions Program Teams: Meet Marcos Prunello and Lukas Wallrich"
author:
  -  Marcos Prunello
  - Lukas Wallrich
date: '2023-04-18'
tags:
  - community
  - champions program
featured: true
description: "Introducing Champions and Mentors - Marcos Prunello and Lukas Wallrich"
---

*We designed the [rOpenSci Champions Program](/champions/) with a mentorship aspect. Mentoring plays a significant role in the growth and development of both mentors and mentees alike. In our program, each Champion has a mentor who accompanies them during their training and development of their project.*

*In this series of blog posts, we introduce you to the ten teams of this first cohort and what they will be working on in the program.*

Meet **Marcos Prunello** from Argentina and **Lukas Wallrich** from United Kingdom!

## Marcos Prunello - Champion

{{< figure src = "/img/team/marcos-prunello-mentee.jpg" class = "pull-left" width = "300" alt = "Profile photo of Marcos Prunello" caption = "<center><strong>Marcos Prunello</strong></center>">}}

> 

Author: ```suggestion
My name is Marcos Prunello and I’m from Argentina. I have a Bachelor of Science in Statistics from Universidad Nacional de Rosario (UNR) and an Academic Master of Science from Stanford University. I am currently working full time as a statistician and professor at UNR, where I have been teaching Statistics and Data Science for several years, training students in topics related to programming, exploratory data analysis, numerical methods and non-parametric statistics.  I am also a co-founder of the R User Group in my city "R in Rosario" and I am frequently invited to different places to offer workshops on the use of R. 

As a work criterion, I have chosen to teach my classes and produce my study materials using R. In doing so I grew interested in being part of the community of developers that make data analysis tools available for everyone and developed some packages that are now in CRAN (“ggcleveland”, “geneticae”). However, I recently worked on a project that allowed me to combine my interest in teaching and software development: a bilingual package called karel that aims to teach general notions of programming in R in an interactive way for people without any kind of knowledge. Considering that education is a pillar for the construction of reproducible and open research, I applied to ROpenSci’s Champions Program to submit this package for discussion and review. I consider that this will be a good opportunity to get involved in a community of practice as a continuation of the path I am following and with the certainty that the experience acquired will have a positive impact in my work environment.
```

Reviewer: Looks fine to me! Thanks!

</br>

## Lukas Wallrich - Mentor

{{< figure src = "/img/team/lukas-mentor.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Lukas Wallrich" caption = "<center><strong>Lukas Wallrich</strong></center>">}}

> 

Author: ```suggestion
I am Lukas Wallrich - originally from Germany - and work as a lecturer (associate professor) in Organisational Psychology at Birkbeck, University of London, in the UK. My academic research focuses on the opportunities that come with workforce diversity, as well as on the adoption of Open Science practices - and these practices, together with my passion for R, have shaped much of my research methods teaching. In addition, I enjoy developing tools that support reproducible science. I have authored two R packages (rsprite2 and timesaveR) and contributed to several others. As a mentor on the Champion Program, I hope to pass on some of my knowledge and experience, get to know new members of the R community, and support the development of exciting new tools ... I can't wait for the journey to start :)
```

> We're excited to have both Pao and Adam be part of the Champions program and can't wait to see what this partnership yields. Stay-tuned for the next series of introductions 😉 .

Author: ```suggestion
We're excited this partnership already is working on Marcos' project.  Be sure to don't miss the next series of introductions 😉 .
```

</br>

## Lukas Wallrich - Mentor

{{< figure src = "/img/team/lukas-mentor.jpeg" class = "pull-left" width = "300" alt = "Profile photo of Lukas Wallrich" caption = "<center><strong>Lukas Wallrich</strong></center>">}}

> I am Lukas Wallrich - originally from Germany - and work as a lecturer (associate professor) in Organisational Psychology at Birkbeck, University of London, in the UK. My academic research focuses on the opportunities that come with workforce diversity, as well as on the adoption of Open Science practices - and these practices, together with my passion for R, have shaped much of my research methods teaching. In addition, I enjoy developing tools that support reproducible science. I have authored two R packages (rsprite2 and timesaveR) and contributed to several others. As a mentor on the Champion Program, I hope to pass on some of my knowledge and experience, get to know new members of the R community, and support the development of exciting new tools ... I can't wait for the journey to start :)

Author: ```suggestion
I am Lukas Wallrich - originally from Germany - and work as a lecturer (associate professor) in Organisational Psychology at Birkbeck, University of London, in the UK. My academic research focuses on the opportunities that come with workforce diversity, as well as on the adoption of Open Science practices - and these practices, together with my passion for R, have shaped much of my research methods teaching. In addition, I enjoy developing tools that support reproducible science. I have authored two R packages (rsprite2 and timesaveR) and contributed to several others. As a mentor on the Champion Program, I hope to pass on some of my knowledge and experience, get to know new members of the R community, and support the development of exciting new tools.
```
